src/aatoolbox/datasources/fewsnet/fewsnet.py

```diff
@@ -27,243 +32,218 @@
+        region_name = self._country_config.fewsnet.region_name
+        region_code = _REGION_NAME_CODE_MAPPING[region_name]
+        url_region_date = _BASE_URL_REGION.format(
+            region_code=region_code,
+            region_name=region_name,
+            YYYY=date_pub.year,
+            MM=f"{date_pub.month:02d}",
+        )
 
-    """
-    # create tempdir to write zipfile to
-    with TemporaryDirectory() as temp_dir:
-        zip_path = Path(temp_dir) / zip_filename
-        download_url(url=url, save_path=zip_path)
-        logger.info(f"Downloaded {url} to {zip_path}")
-
-        try:
-            unzip(zip_file_path=zip_path, save_dir=output_dir)
-            logger.debug(f"Unzipped to {output_dir}")
-
-        except zipfile.BadZipFile:
-            # indicates that the url returned something that wasn't a
-            # zip, happens often and indicates data for the given
-            # country - date is not available
-            logger.debug(
-                f"No zip data returned from url {url} "
-                f"check that the area and date exist."
+        output_dir_region = (
+            self._raw_base_dir / f""
+            f"{region_code}"
+            f"{date_pub.strftime('%Y%m')}"
+        )
+        zip_filename_region = f"{output_dir_region.name}.zip"
+        if not output_dir_region.exists() or clobber is True:
+            region_data = self._download_zip(
+                url=url_region_date,
+                zip_filename=zip_filename_region,
+                output_dir=output_dir_region,
             )
-            return None
-
-    return output_dir
+        else:
+            region_data = output_dir_region
+        return region_data
+
+    def _download_zip(
+        self,
+        url: str,
+        zip_filename: str,
+        output_dir: Path,
+    ) -> Optional[Path]:
+        """
+        Download and unzip the file at the url.
+
+        Parameters
+        ----------
+        url : str
+            url that contains the zip file to be downloaded
+        zip_filename : str
+            name of the zipfile
+        output_dir : Path
+            path of dir to which the zip content should be written
+
+        Returns
+        -------
+        None if no valid file, else output_dir
+
+        """
+        # create tempdir to write zipfile to
+        with TemporaryDirectory() as temp_dir:
+            zip_path = Path(temp_dir) / zip_filename
+            download_url(url=url, save_path=zip_path)
+            logger.info(f"Downloaded {url} to {zip_path}")
+
+            try:
+                unzip(zip_file_path=zip_path, save_dir=output_dir)
+                logger.debug(f"Unzipped to {output_dir}")
+
+            except zipfile.BadZipFile:
+                # indicates that the url returned something that wasn't a
+                # zip, happens often and indicates data for the given
+                # country - date is not available
+                logger.debug(
```

Comment:
As mentioned above, I would re-raise this error and catch it in the case of `country` so that you can try the region.